unittests/Feature/FeatureModelSxfm.cpp

```diff
@@ -12,17 +12,33 @@ namespace vara::feature {
 TEST(SxfmParser, validation) {
```

Comment:
Maybe add a short description to each test what the test wants to test

Reply:
This is done in the next commit. Thanks!